lib/paginator/ecto/query/asc_nulls_first.ex

```diff
@@ -0,0 +1,68 @@
+defmodule Paginator.Ecto.Query.AscNullsFirst do
+  @behaviour Paginator.Ecto.Query.DynamicFilterBuilder
+
+  import Ecto.Query
+
+  @impl Paginator.Ecto.Query.DynamicFilterBuilder
+  def build_dynamic_filter(:after, nil) do
+    fn
+      _, _, true ->
```

Comment:
I wonder if there's a benefit to returning an anonymous function here rather than building and returning the filter in one step. From what I can tell the anonymous functions are always invoked as soon as they are built.

Reply:
👍🏽 good point, we don't need the anonymous function here. I'm removing it.

Reply:
done